lib/log/appender/banner.js

```diff
@@ -2,12 +2,17 @@ var insertCSS = require('../../style/insertCss.js');
 var outDiv = null;
 var startTime = new Date().getTime();
 var OUTPUT_ID = 'logoutput';
+var overideTimeout;
+
+function timeoutWrap(fn, ms){
```

Comment:
Same as for index.js. We should try to avoid test-specific code outside the tests.